src/util/util.ts

```diff
@@ -98,3 +98,17 @@ function parseNftTokenUriArray(arr: TokenUri[] | undefined): TokenUri[] {
   }
   return arr.filter(uri => parseNftTokenUri(uri) !== undefined);
 }
+
+export function stripUndefined(obj: { [key: string]: any }): {
```

Comment:
Can we add a comment on why this is necessary (either here or in the usage)? The low-level code is a little hard to read without context.

Reply:
done.